tests/integration/webclient/src/main/java/io/helidon/tests/integration/webclient/Main.java

```diff
@@ -82,20 +84,17 @@ private static CompletionStage<WebServer> startIt(Config config, WebServer.Build
         // print a message at shutdown. If unsuccessful, print the exception.
         CompletionStage<WebServer> start = webServer.start();
 
-        start.thenAccept(ws -> {
+        return webServer.start()
+                .peek(ws -> {
                     serverPort = ws.port();
                     System.out.println(
                             "WEB server is up! http://localhost:" + ws.port() + "/greet");
                     ws.whenShutdown().thenRun(() -> System.out.println("WEB server is DOWN. Good bye!"));
                 })
-                .exceptionally(t -> {
+                .onError(t -> {
                     System.err.println("Startup failed: " + t.getMessage());
                     t.printStackTrace(System.err);
-                    return null;
                 });
```

Comment:
orError doesn't get executed if single is ignored

Reply:
Ignoring elements from main, await in test.